pages/contributors.js

```diff
@@ -143,47 +143,21 @@ export default function Contributors({ contributors }) {
 Contributors.propTypes = {
   contributors: PropTypes.arrayOf(
     PropTypes.shape({
+      name: PropTypes.string,
       login: PropTypes.string.isRequired,
-      id: PropTypes.number.isRequired,
-      node_id: PropTypes.string.isRequired,
+      twitter_username: PropTypes.string,
       avatar_url: PropTypes.string.isRequired,
-      gravatar_id: PropTypes.string,
-      url: PropTypes.string.isRequired,
+      blog: PropTypes.string,
       html_url: PropTypes.string.isRequired,
-      followers_url: PropTypes.string.isRequired,
-      following_url: PropTypes.string.isRequired,
-      gists_url: PropTypes.string.isRequired,
-      starred_url: PropTypes.string.isRequired,
-      subscriptions_url: PropTypes.string.isRequired,
-      organizations_url: PropTypes.string.isRequired,
-      repos_url: PropTypes.string.isRequired,
-      events_url: PropTypes.string.isRequired,
-      received_events_url: PropTypes.string,
-      type: PropTypes.string.isRequired,
-      site_admin: PropTypes.bool.isRequired,
       contributions: PropTypes.number.isRequired,
-      name: PropTypes.string.isRequired,
-      company: PropTypes.string,
-      blog: PropTypes.string,
-      location: PropTypes.string,
-      email: PropTypes.string,
-      hireable: PropTypes.bool,
-      bio: PropTypes.string,
-      twitter_username: PropTypes.string,
-      public_repos: PropTypes.number.isRequired,
-      public_gists: PropTypes.number.isRequired,
-      followers: PropTypes.number.isRequired,
-      following: PropTypes.number.isRequired,
-      created_at: PropTypes.string.isRequired,
-      updated_at: PropTypes.string.isRequired,
     })
   ).isRequired,
 }
 
 export async function getStaticProps() {
   if (!process.env.GITHUB_TOKEN) {
     throw new Error(
-      `Missing the environment variable GITHUB_TOKEN. If this is happening for local development, ensure you've generated a token and included it in your .env file, e.g. GITHUB_TOKEN="some-token". To generate a personal access token, follow the steps in the GitHub documentation, https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token.`
+      `Missing the environment variable GITHUB_TOKEN. If this is happening for local development, ensure you've generated a token and included it in your .env file, e.g. GITHUB_TOKEN="some-token". To generate a personal access token, follow the steps in the GitHub documentation, https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token . For the scope of the token being created, select public_repo only from within the repo scope.`
```

Comment:
Made the error message more clear @sethburtonhall as you weren't sure which scope to add when creating the GitHub token.